src/typval.c

```diff
@@ -450,6 +450,37 @@ check_for_opt_string_arg(typval_T *args, int idx)
 	    || check_for_string_arg(args, idx) != FAIL) ? OK : FAIL;
 }
 
+/*
+ * Check for an optional string argument at 'idx', that can also be 'v:none' to
+ * use the default value.
+ *
+ * If 'is_none' is non-NULL it is set to 0 and updated to 1 when "args[idx]" is
+ * 'v:none'.
+ */
+    int
+check_for_opt_string_or_none_arg(typval_T *args, int idx, int *is_none)
```

Comment:
I used it at first, but then, it turned out to be unnecessary.

It seems that distinguishing between the "value given" and "use default" cases could be important.
So, I figured, the next use of this helper function could actually benefit from the `is_none` flag, and kept it.
Sorry, I should have mentioned it.

I can remove it, if you think that it can be added when an actual need will arise.

Reply:
yeah, that's what I thought. Thanks